frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/emoji_picker/emoji_menu_item.dart

```diff
@@ -82,7 +82,9 @@ class _EmojiSelectionMenuState extends State<EmojiSelectionMenu> {
     if (event.logicalKey == LogicalKeyboardKey.escape &&
         event is KeyDownEvent) {
       //triggers on esc
-      widget.onExit();
+      if (widget.onExit != null) {
+        widget.onExit!();
+      }
```

Comment:
```suggestion
      widget.onExit?.call();
```

Reply:
Oh, my bad, don't know how missed that, thank you. Committed your suggestion